tests/integration/targets/docker_container/tasks/tests/options.yml

```
@@ -1143,6 +1143,66 @@
       ('API version is ' ~ docker_api_version ~ '.') in device_requests_1.msg and 'Minimum version required is 1.40 ' in device_requests_1.msg
   when: docker_api_version is version('1.40', '<')
 
+####################################################################
+## device_cgroup_rules ###################################################
+####################################################################
+
+- name: device_cgroup_rules
+  docker_container:
+    image: "{{ docker_test_image_alpine }}"
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    force_kill: true
+    device_cgroup_rules:
+    - "c 42:* rmw"
+  register: device_cgroup_rules_1
+  ignore_errors: yes
```

```suggestion
  ignore_errors: true
```
Same below, and `no` → `false`.